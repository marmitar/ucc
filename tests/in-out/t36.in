void f() {
  N=N+1;
  N=-N;
  N=N%0;
  N=N-100;
  N=N/4;
}
