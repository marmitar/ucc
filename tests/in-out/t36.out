SemanticError: Binary operator + does not have matching LHS/RHS types @ 2:5
